doc/en/assert.rst

```diff
@@ -246,8 +246,7 @@ the conftest file::
            f1 = Foo(1)
            f2 = Foo(2)
    >       assert f1 == f2
-   E       assert Comparing Foo instances:
-   E            vals: 1 != 2
+   E       AssertionError
```

Comment:
Hmm, this seems like a weird change seeing that the doc is about `pytest_assertrepr_compare`?